docker-compose.yml

```diff
@@ -32,7 +32,7 @@ services:
     volumes:
       - ./services/:/services
     working_dir: /services/registry
-    command: sh -c "npm run prestart && npm run dev"
```

Comment:
This was broken.

Reply:
[My comment about `npm run dev`](https://github.com/entropic-dev/entropic/pull/242#discussion_r292239446)

Reply:
Same change came in in #242